### PR TITLE
Metrics page + Modal Task

### PR DESCRIPTION
**Pull request by Vasugi14**
Tasks completed for this merge:
- Modal added to survey when the survey is not live
- Metrics page created to house components to show MC and Text responses
Check through to see if I should change anything please!

### CODE REVIEW

File: app/javascript/components/MetricsPage.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography'
+import Card from '@mui/material/Card'
+import { createTheme, ThemeProvider, styled } from '@mui/material/styles';
+import SurveyList from './SurveyList';
+import SurveyListActions from './SurveyListActions'
+
+
+const darkTheme = createTheme({ palette: { mode: 'dark' } });
```
Comment (AashnaNarang) [inline app/javascript/components/MetricsPage.js:12]: this stuff isnt needed and some of the imports are not needed, we should remove those

File: app/javascript/components/MetricsPage.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    const checkRequest = (res) => {
+        if (res.status === 200) {
+            return res.json();
+        } else {
+            throw res;
+        }
+    }
+
+    useEffect(async () => {
+        console.log("checker 2")
```
Comment (AashnaNarang) [inline app/javascript/components/MetricsPage.js:29]: remove console.log statements

File: app/javascript/components/MetricsPage.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    // return(
+    //     <div className="surveyResponses">
+    //         <Card>
+    //             <ThemeProvider theme={darkTheme}>
+    //                 <Typography variant="h4" component="h3">
+    //                     Hello surveyor, Welcome to the Metrics Page!
+    //                 </Typography> 
+    //                 <Paper variant="outlined" />
+    //             </ThemeProvider>
```
Comment (AashnaNarang) [inline app/javascript/components/MetricsPage.js:53]: remove this too

File: app/javascript/components/MetricsPage.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    //         </Card>
+    //     </div>
+    // )
+
+    return(
+        <div className="surveyResponses">
+            <Typography variant="h2">{title}</Typography>
+            <br/>
+            <br/>
+
```
Comment (AashnaNarang) [inline app/javascript/components/MetricsPage.js:65]: remove the extra br tags

File: app/javascript/components/Survey.js [hunk 5]
```diff
@@ -31,17 +35,30 @@ const Survey = () => {
...
             },
         })
         .then(checkRequest)
-        .then(data => {
+        .then(async (data) => {
+            console.log(data.survey.isLive)
+            if(!data.survey.isLive) {
+                handleOpen();
+            }
+            await createResponder();
```
Comment (AashnaNarang) [inline app/javascript/components/Survey.js:61]: to be safe, the next three lines should be in an else statement, also remove the console.log on line 57

**Comment by Vasugi14** [comment]
Made changes to code based on review from above, if its good I can merge it.

**Review by AashnaNarang — APPROVED** [review]

**Pull request merged**